base/common/python/pki/nssdb.py

```diff
@@ -273,21 +273,24 @@ def dbexists(filename):
             return os.path.isfile(os.path.join(self.directory, filename))
 
         if dbexists('cert9.db'):
-            if not dbexists('key4.db') or not dbexists('pkcs11.txt'):
+            if not dbexists('key4.db'):
                 raise RuntimeError(
                     "{} contains an incomplete NSS database in SQL "
                     "format".format(self.directory)
                 )
-            return 'sql'
-        elif dbexists('cert8.db'):
+
+            # secmod.db is not auto converted at NSS_INIT_NOMODDB
+            if dbexists('pkcs11.txt'):
+                return 'sql'
```

Comment:
If the cert9.db and key4.db exist, but pkcs11.txt is missing, this code will return None. Is that the intended behavior? Or should it throw an exception?